test/conditionals_test.dart

```diff
@@ -373,6 +392,97 @@ void main() {
           signalToWidthMap: {'d': 8, 'q': 8});
       expect(simResult, equals(true));
     });
+
+    test('should return exception if a conditional is used multiple times.',
+        () async {
+      expect(
+          () => MultipleConditionalModule(Logic(), Logic()), throwsException);
+    });
+  });
+
+  test(
```

Comment:
Why are all these tests added with this PR?  Were they missing last PR?

Reply:
It is because of the merge conflict. Some spacing added. So, it treats it like a newly added line since I add the conditionals test on the middle.

Reply:
It looks like the conflict resolution actually added duplicates to all these tests?  You should remove these duplicates

Reply:
Removed duplicated code